src/Mailgun/Exception.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Mailgun;
+
+/**
+ * All Mailgun exception implements this exception
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+interface Exception
```

Comment:
There is a good idea to have a base exception. The one who is lazy may just catch this exception and he/she knows that something went wrong